### ISSUE TITLE
annotation_layer_builder.css is not included in the build

### ISSUE THREAD

**Issue by flacerdk** [wojtekmaj/react-pdf#87]
As a result, importing `Page` results in the error:

`Module not found: Can't resolve './annotation_layer_builder.css' in 'node_modules/react-pdf/build'`

A workaround for people using webpack with `css-loader` is to copy the file from `src/` to `build/`. However this won't work in setups that don't resolve CSS files.

A solution here is to bundle the source together using webpack (as in [here](https://webpack.js.org/guides/author-libraries/)), using `css-loader` to resolve the CSS.

**Comment by wojtekmaj** [comment]
Oh, damn. Thanks for reporting! Pushing the fix right away.

**Comment by wojtekmaj** [comment]
Fixed in 2.2.0-beta.2

**Comment by JRigotti** [comment]
I am running the `2.4.1` and just got the same error:

```
ERROR in ./node_modules/react-pdf/build/annotation_layer_builder.css
Module parse failed: Unexpected token (16:0)
You may need an appropriate loader to handle this file type.
|  */
|
| .annotationLayer section {
|   position: absolute;
| }
 @ ./node_modules/react-pdf/build/PageAnnotations.js 35:0-41
 @ ./node_modules/react-pdf/build/Page.js
 @ ./node_modules/react-pdf/build/entry.js
 @ ./resources/assets/redux/components/PDFViewer.jsx
```

Which workaround can I use it to make it work?

**Comment by wojtekmaj** [comment]
Actually you did not! Your problem is that your Webpack config lacks any handling of CSS files, one of which included by React-PDF for handling annotations.

**Comment by iamrommel** [comment]
My webpack config was to exclude the node_modules folder for css loader, then how should it work with this css

**Comment by wojtekmaj** [comment]
@iamrommel, your problem is actually quite popular and I addressed it React-PDF 3.0 which is nearing release. In React-PDF 3.0, you will need to manually import CSS styles for annotations should you want to use them (or, of course, create your own ones).

You can help with getting React-PDF ready for its prime time in #143 :)

**Comment by iamrommel** [comment]
@wojtekmaj , thanks, the version works for me now. I
